### ISSUE TITLE
[Feature] Add option for check update if extension installed

### ISSUE THREAD

**Issue by jhen0409** [MarshallOfSound/electron-devtools-installer#28]
Add option for check update if extension installed, I think it will be better than `forceDownload`.

For `redux-devtools-extension`, we can got the xml by `https://clients2.google.com/service/update2/crx?x=id%3Dlmhkpmbekcpmknklioeibfkpmmfibljd%26uc&prodversion=32` (No `response`):

```xml
<gupdate xmlns="http://www.google.com/update2/response" protocol="2.0" server="prod">
  <daystart elapsed_days="3620" elapsed_seconds="67699"/>
  <app appid="lmhkpmbekcpmknklioeibfkpmmfibljd" cohort="" cohortname="" status="ok">
    <updatecheck
      codebase="https://clients2.googleusercontent.com/crx/blobs/QgAAAC6zw0qH2DJtnXe8Z7rUJP2IRVE-L9w2IT8562wiulh2IgB09conu2XJ-g73l8oozDkTpsK4nut8cp-7Murq2_r8MbzmmbAhUzPkQuuESgjcAMZSmuV2YJ8g9OU9pR7i6f1D9n90dSy57w/extension_2_11_0_4.crx"
      fp="1.48a24cd00dc54fb49dde23bb80d57eeb2dd71a6380572ca5cad9831b60d36fd0"
      hash_sha256="48a24cd00dc54fb49dde23bb80d57eeb2dd71a6380572ca5cad9831b60d36fd0"
      protected="0"
      size="1486330"
      status="ok"
      version="2.11.0.4"
    />
  </app>
</gupdate>
```

**Comment by fomenyesu** [comment]
agree ! I already installed all the extensions but still got the error msg..

**Comment by SimulatedGREG** [comment]
Would love to have some sort of auto-update feature. I put this [commit](https://github.com/SimulatedGREG/electron-devtools-installer/commit/006ed428040467dc5ee83903ca4b92e722416c66) together but soon realized @jhen0409 strategy doesn't seem to work for all extensions unfortunately. Will look into later.

**Comment by jhen0409** [comment]
@SimulatedGREG Thanks for do this! I haven't enough time to.

Look at your commit, `semver` can't compatible Chrome extension version, like currently `redux-devtools-extension` version is `2.12.2.1`, should we update the extension just need to know the version is different?

**Comment by SimulatedGREG** [comment]
Yeah I noticed that afterwards, although VUEJS_DEVTOOLS actually does use proper `semver` with version `3.0.5`. Either way, when you plugin in the chromeStoreId for each plugin, not all produce a version number.
